orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/OrcaMessageHandler.kt

```diff
@@ -64,6 +64,10 @@ internal interface OrcaMessageHandler<M : Message> : MessageHandler<M> {
       try {
         execution
           .stageById(stageId)
+          .also {
+            // Mutates it.context in a required way for some non-linear stage features
```

Comment:
I wasn't sure what the "required way" meant here. Maybe worth expanding the comment a little bit?